packages/@aws-cdk/aws-amplify/test/integ.app.ts

```diff
@@ -25,14 +25,17 @@ class TestStack extends Stack {
           },
         },
       ],
+      platform: amplify.Platform.WEB,
     });
 
     amplifyApp.addCustomRule({
       source: '/source',
       target: '/target',
     });
 
-    const mainBranch = amplifyApp.addBranch('main');
+    const mainBranch = amplifyApp.addBranch('main', {
```

Comment:
This should be its own test, not just an update to a current test. Also, we should be using the new `IntegTest` construct for all new or edited test cases. I'd like to see an assertion added to the new test to ensure that it actually successfully deploys as well.

Reply:
Agreed, this needs to be a separate `IntegTest`.